docker-compose.yaml

```diff
@@ -0,0 +1,73 @@
+version: '3.1'
+
+services:
+
+  mysql:
+    image: mysql
+    restart: always
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      MYSQL_ROOT_PASSWORD: root # Change Me
+      MYSQL_DATABASE: otservbr-global # Change Me
+      MYSQL_USER: otservbr-global # Change Me
+      MYSQL_PASSWORD: otservbr-global # Change Me
+    ports:
+      - 3306:3306
+    volumes:
+      - canary-db:/var/lib/mysql
+      - ./schema.sql:/docker-entrypoint-initdb.d/schema.sql
+    networks:
+      - canary
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
+    networks:
+      - canary
+    depends_on:
+      - mysql
+
+  server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: always
+    ports:
+      - 7171:7171
+      - 7172:7172
+    volumes:
+      - ./data:/srv/canary/data
+      - ./data-canary:/srv/canary/data-canary
+      - ./data-otservbr-global:/srv/canary/data-otservbr-global
+      - ./config.lua:/srv/canary/config.lua
+    networks:
+      - canary
+    depends_on:
+      - mysql
+
+  # pre-built-server:
```

Comment:
Remove commented code

Reply:
This is just an example of how to run the `pre-built-server` instead of building the image like it's done with the `server` service. They shouldn't run at the same time, so I would rather remove it them uncomment it.

Reply:
Maybe you can add some comments here to explain that (I didn't checked if you made this on the documentation, if yes so you can ignore this comment)